examples/elasticsearch/es-rc.yaml

```diff
@@ -48,5 +49,3 @@ spec:
           name: storage
       volumes:
       - name: storage
-        source:
-          emptyDir: {}
```

Comment:
The README has a big notice about the mapping for `emptyDir`; Kubernetes 1.1+ doesn't like this. What implications does that have for the README